### PR TITLE
stop using throw

### PR DESCRIPTION
**Pull request by bollwyvl**
## References

- attempted fix for part of #299
- ~~may investigate adding #130~~

## Code changes

- [x] removes all custom `throws`, instead returning `null` (most of these were already sometimes returning `null`)

### not on this PR
- [ ] ~~take this opportunity to enable `strictNullChecks` and clean up some more return types~~

## User-facing changes

- should not result in failed mousemove handler calls

## Backwards-incompatible changes

- still looking into this... behavior was probably just unspecified

## Chores

- [x] linted
- [ ] tested
- [ ] documented
- [x] changelog entry

### CODE REVIEW

**Comment by bollwyvl** [comment]
I had a look at #130 I think it's too much to do here. Indeed, we will likely want to go package-by-package rather than all in one go.

**Comment by bollwyvl** [comment]
I'm not really sure how to test this properly... it seems the plumbing underneath the other extension is rather heavyweight, and doesn't have an existing binder.

File: docs/Installation.ipynb [hunk 1]
```diff
@@ -8,15 +8,27 @@
    },
    "outputs": [],
    "source": [
-    "from markdown import markdown\n",
     "import sys\n",
-    "sys.path.insert(0, '..')\n",
+    "\n",
+    "from markdown import markdown\n",
+    "\n",
+    "sys.path.insert(0, \"..\")"
```
Comment (krassowski) [inline docs/Installation.ipynb:15]: Good one!

**Comment by bollwyvl** [comment]
Hooray!

**Pull request merged**